<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcesGraphEditor</class>
 <widget class="QMainWindow" name="ResourcesGraphEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resources Graph Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pics/pics/graph.png</normaloff>:/pics/pics/graph.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QMenuBar::item {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #c5d8ef, stop: 1 #89a5c3);
}

QLabel, QSlider {
	background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="graphArea">
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>398</width>
           <height>230</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_System"/>
    <addaction name="actionOpen_System"/>
    <addaction name="actionSave_System"/>
    <addaction name="actionSave_System_As"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_System"/>
   <addaction name="actionOpen_System"/>
   <addaction name="actionSave_System"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionComputer"/>
   <addaction name="actionStorage"/>
   <addaction name="actionRouter"/>
   <addaction name="actionEdge"/>
   <addaction name="separator"/>
   <addaction name="actionTopology"/>
  </widget>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/select.png</normaloff>:/pics/pics/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionComputer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/computer.png</normaloff>:/pics/pics/computer.png</iconset>
   </property>
   <property name="text">
    <string>Add Computational Node</string>
   </property>
   <property name="toolTip">
    <string>Add Computational Node</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionEdge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/edge.png</normaloff>:/pics/pics/edge.png</iconset>
   </property>
   <property name="text">
    <string>Add Channel</string>
   </property>
   <property name="toolTip">
    <string>Add Channel</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionNew_System">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/page.png</normaloff>:/pics/pics/page.png</iconset>
   </property>
   <property name="text">
    <string>New Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_System">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/folder.png</normaloff>:/pics/pics/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_System">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/cd.png</normaloff>:/pics/pics/cd.png</iconset>
   </property>
   <property name="text">
    <string>Save Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_System_As">
   <property name="text">
    <string>Save Graph As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionStorage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/storage.png</normaloff>:/pics/pics/storage.png</iconset>
   </property>
   <property name="text">
    <string>Add Data Store</string>
   </property>
   <property name="toolTip">
    <string>Add Data Store</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionRouter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/router.png</normaloff>:/pics/pics/router.png</iconset>
   </property>
   <property name="text">
    <string>Add Commutation Element</string>
   </property>
   <property name="toolTip">
    <string>Add Commutation Element</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionTopology">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/pics/pics/topology.png</normaloff>:/pics/pics/topology.png</iconset>
   </property>
   <property name="text">
    <string>Generate Standart Topology</string>
   </property>
   <property name="toolTip">
    <string>Generate Standart Topology</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSelect</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>toggleSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionComputer</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>toggleComputer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdge</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>toggleEdge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_System</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>New()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_System</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_System</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_System_As</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>SaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStorage</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>toggleStorage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRouter</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>toggleRouter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTopology</sender>
   <signal>triggered()</signal>
   <receiver>ResourcesGraphEditor</receiver>
   <slot>generateTopology()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleSelect()</slot>
  <slot>toggleComputer()</slot>
  <slot>toggleEdge()</slot>
  <slot>New()</slot>
  <slot>Open()</slot>
  <slot>Save()</slot>
  <slot>SaveAs()</slot>
  <slot>toggleStorage()</slot>
  <slot>toggleRouter()</slot>
  <slot>generateTopology()</slot>
 </slots>
</ui>
